packages/web/src/scss/components/Button/_Button.scss

```diff
@@ -46,19 +53,14 @@
     );
 
     flex: none;
-    padding-left: 0;
-    padding-right: 0;
+    padding-inline: 0;
 }
 
 .Button--block {
     display: block;
     width: 100%;
 }
 
-@each $size, $variables in theme.$sizes {
-    @include tools.size($size, $variables);
-}
-
-@each $variant, $variables in theme.$variants {
-    @include tools.variant($variant, $variables);
+.Button--square.Button--large {
+    padding-inline: map.get(theme.$sizes, medium, padding-x);
```

Comment:
`medium` size padding for a `large` button? Maybe there's another way? 🙂

Reply:
🙈 Of course there is!